docs/Cli.md

```diff
@@ -133,6 +133,7 @@ Re-apply any new defaults as desired.
 | `auto_disarm_delay`             |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | 0      | 60     | 5             | Master       | UINT8    |
 | `small_angle`                   | If the copter tilt angle exceed this value the copter will refuse to arm. default is 25°.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 180    | 25            | Master       | UINT8    |
 | `pid_at_min_throttle`           | If enabled, the copter will process the pid algorithm at minimum throttle.  Cannot be used when `retarded_arm` is enabled.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             | OFF    | ON     | ON            | Master       | UINT8    |
+| `airmode_saturation_limit`      | When air mode is enabled this will act as a "crash detection". It prevents too much compensation for very high errors what can cause spazzing out. 0 = no limit 100 is always limited during saturation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              | 0      | 100    | 50            | Master       | UINT8    |
```

Comment:
inverted value (with some scaling) would make more sense - 200% limit would mean to allow compensation untill requested PRY is smaller than 200% throttleRange